apps/duellen/app/components/Question.jsx

```diff
@@ -91,23 +91,43 @@ export default class Question extends React.Component {
       userInput: '',
     })
   };
+  handleSkip(e){
+    e.preventDefault();
+    this.hideMessage()
+    this.setState({message: cogoToast.info('Frågan skippad', {toastContainerID: '1'})})
+    this.handleAnswer(e);
+    this.setState({
+      userInput: '',
+    })
+  };
 
   handleWrongRight(e){
     e.preventDefault();
     const {tally, hintPoint} = this.state;
     if(this.state.userInput === this.checkIfCorrect()){
-      cogoToast.success('Du fick den rätt');
-      this.setState({tally: tally + hintPoint});
+      this.hideMessage()
+      var test = cogoToast.success('Du fick den rätt', {toastContainerID: '1'});
```

Comment:
Let's rename the `test` variable to something more meaningful (unless it really is a test of some sort :D)